packages/js/manifests/polywrap/src/formats/polywrap.deploy/0.3.0.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface DeployManifest {
+  /**
+   * Polywrap deployment manifest format version.
+   */
+  format: "0.3.0";
+  /**
+   * Name of the job that runs the primary deployment sequence.
+   */
+  primaryJob?: string;
```

Comment:
Would it make more sense to call this `primaryJobName` for the sake of clarity? Just a nitpick!

Reply:
Good call!